From foo@baz Mon Jan  5 11:22:07 PST 2009
Date: Mon, 05 Jan 2009 11:22:07 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: go7007: add some more v4l2 ioctls

Now that the v4l2 tree is merged with mainline, enable some of
the compiled out functions.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/go7007/go7007-v4l2.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/staging/go7007/go7007-v4l2.c
+++ b/drivers/staging/go7007/go7007-v4l2.c
@@ -1091,7 +1091,6 @@ static int vidioc_s_parm(struct file *fi
 
    The two functions bellow implements the newer ioctls
 */
-#if 0
 static int vidioc_enum_framesizes(struct file *filp, void *priv,
 				  struct v4l2_frmsizeenum *fsize)
 {
@@ -1133,7 +1132,6 @@ static int vidioc_enum_frameintervals(st
 
 	return 0;
 }
-#endif
 
 static int vidioc_s_std(struct file *file, void *priv, v4l2_std_id *std)
 {
@@ -1819,10 +1817,8 @@ static const struct v4l2_ioctl_ops video
 	.vidioc_s_frequency       = vidioc_s_frequency,
 	.vidioc_g_parm            = vidioc_g_parm,
 	.vidioc_s_parm            = vidioc_s_parm,
-#if 0  /* FIXME take out after 2.6.29-rc1 merge happens */
 	.vidioc_enum_framesizes   = vidioc_enum_framesizes,
 	.vidioc_enum_frameintervals = vidioc_enum_frameintervals,
-#endif
 	.vidioc_cropcap           = vidioc_cropcap,
 	.vidioc_g_crop            = vidioc_g_crop,
 	.vidioc_s_crop            = vidioc_s_crop,
